notifications/static/notifications/notify.js

```diff
@@ -0,0 +1,63 @@
+var notify_badge_id;
+var notify_menu_id;
+var notify_api_url;
+var notify_unread_url;
+var notify_mark_all_unread_url;
+var notify_refresh_period = 15000;
+var consecutive_misfires = 0;
+var registered_functions = [];
+
+function fill_notification_badge(data) {
+    var badge = document.getElementById(notify_badge_id);
+    if (badge) {
+        badge.innerHTML = data.unread_count;
+    }
+}
+
+function fill_notification_list(data) {
+    var menu = document.getElementById(notify_menu_id);
+    if (menu) {
+        menu.innerHTML = "";
+        for (var i=0; i < data.unread_list.length; i++) {
+            var item = data.unread_list[i];
+            menu.innerHTML = menu.innerHTML + "<li>"+item.object+" "+item.verb+" "+item.subject+"</li>";
```

Comment:
I know this has already been merged, but where are `item.object` and `item.subject` in this repo's standard JSON response? Those aren't available in `data.unread_list` items AFAICT.

I'm testing out this repo right now and it's falling down here.

Screenshots may help:

![screenshot 2015-11-10 16 57 21](https://cloud.githubusercontent.com/assets/344623/11080789/44ee6478-87cc-11e5-8339-a39cb7f46fc7.png)

![screenshot 2015-11-10 16 58 17](https://cloud.githubusercontent.com/assets/344623/11080791/4acb6558-87cc-11e5-812a-fe584b82db79.png)